taskgraph/Task.py

```diff
@@ -384,16 +383,14 @@ def __del__(self):
             # it's possible the global state is not well defined, so just in
             # case we'll wrap it all up in a try/except
             self._terminated = True
-            if self._n_workers > 0:
-                LOGGER.debug("shutting down workers")
+            if self._executor_ready_event:
+                # alert exeuctors to check that _terminated is True

```

Comment:
This is trivial, but there's a typo in `exeuctors`.

Reply:
87beca78a86067da5c1c9b9d3067a73c143b88cf